src/utils/AnimationUtils.ts

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2021 Šimon Brandner <simon.bra.ag@gmail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+export function lerp(start: number, end: number, amt: number) {
```

Comment:
Could you maybe add a JSDoc comment explaining what this function achieves?
It should be fairly achievable to add some tests here too

Reply:
I didn't add any tests because it seemed a little weird to me, I don't really expect anyone to make changes to this method, so I am not really sure what would they exactly be for. That being said, I am not the most experienced with writing tests, so I might be overlooking something

Reply:
(or overlooking something not directly related to tests)

Reply:
I don't really see an obvious reason why someone might change this in the future, but this might be a good way to document how this function will behave in certain odd scenario. 

Will it throw? or return an impossible value? ... that probably won't be a blocker for this PR. Let me know how you want to proceed

Reply:
Good point! Will add tests